lldb/include/lldb/Utility/GPUGDBRemotePackets.h

```diff
@@ -190,6 +190,10 @@ struct GPUActions {
   /// Set this to true if the native plug-in sync with the GPU process and wait
   /// for it to return to a running state.
   bool wait_for_gpu_process_to_resume = false;
+  /// Set this to true if the native plug-in should wait synchronously for the
+  /// GPU process to initialize. This should be true if
+  /// `wait_for_gpu_process_to_resume` is set to true as part of initialization.
```

Comment:
Can we add an assert or some error checking to validate the comment here (i.e. that if `wait_for_gpu_process_to_resume == true` then `wait_synchronously_for_gpu_to_initialize == true`).

Reply:
This should be part of the `GPUPluginConnectionInfo` structure and should added as:
```
/// Synchronously wait for the GPU to initialize when connecting.
bool synchronous;
```

Reply:
@dmpots , this time I won't write the assert. GPUActions in general are not waiting yet for the process to be stopped before performing their actions, so I'll wait for GPUActions to implement that instead of writing that assert.